test/puppetlabs/comidi_test.clj

```diff
@@ -271,14 +326,26 @@
 
     (handler {:uri "/bar/something" :request-method :get})
     (is (= (-> test-atom deref :route-info :route-id) "bar-:bar"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/false" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "false"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/true" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "true"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/wat" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "*"))
```

Comment:
So, I see 4 new routes and 3 new tests.  Is it possible to hit the `false` route at all?  I'm not sure I understand what the point of that is :)  (in bidi, I mean, not necessarily here.)